app/src/main/java/net/gsantner/markor/ui/SearchOrCustomTextDialogCreator.java

```diff
@@ -125,11 +125,17 @@ public static void showInsertTableRowDialog(final Activity activity, final boole
 
     public static void showSearchFilesDialog(Activity activity, File searchDir, Callback.a2<String, Integer> callback) {
         if (!SearchEngine.isSearchExecuting) {
+            final char[] password;
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M) {
+                password = new AppSettings(activity).getDefaultPassword();
+            } else {
+                password = new char[0];
+            }
             Callback.a1<SearchEngine.SearchOptions> fileSearchDialogCallback = (searchOptions) -> {
                 searchOptions.rootSearchDir = searchDir;
                 SearchEngine.queueFileSearch(activity, searchOptions, searchResults -> {
                     FileSearchResultSelectorDialog.showDialog(activity, searchResults, callback);
-                });
+                }, password);
```

Comment:
there is a data container searchOptions which should fit instead of creating an specific additional parameter

Reply:
Good Point. I fixed it.